apigateway/helm/templates/kibana.yaml

```diff
@@ -114,10 +119,11 @@ spec:
                   key: password
             {{- end }}
           readinessProbe:
-            httpGet:
-              path: /status
-              port: 5601
-              scheme: HTTP
+            {{- toYaml .Values.kibana.readinessProbe | nindent 12 }}
```

Comment:
{{- end }}

Reply:
?

Reply:
Please make it equal to all probs:
{{- if .Values.kibana.xxxxProbe}}
{{- toYaml .Values.kibana.xxxxProbe | nindent 12 }}
{{- end }}